proto/nodeinstance/info/v1/info.proto

```diff
@@ -45,12 +45,9 @@ message UpdateNodeInfo {
   // node (child) might be available/enabled on the node (child) directly
   // but not available trough the parent (node_instance)
 
-  NodeInstanceInfo node_instance_info = 10;
-  // info specific to the node_instance for this node available trough agent
-  // who sends this message.
-  // e.g. machine learning is enabled for this node and processing is done
-  // by the actual agent (parent). (child itself might or might not be
-  // ml ml_capable by itself (see node_info))
+  //NodeInstanceInfo node_instance_info = 10;
+  // moved to UpdateNodeInstanceConnection
+  reserved 10;
```

Comment:
Why remove it from here?

Reply:
Should it be in both places? Seems like duplicate to me... I can put it back

Reply:
Yeap I suggest to put it back for now, should assess if other stuff would misbehave if this is absent which is something we cannot deal with currently.

Reply:
ok, will do it
my general feeling is however we are adding workaround on top of workaround